src/app-layout/visual-refresh-toolbar/skeleton/index.tsx

```diff
@@ -57,6 +59,7 @@ export function SkeletonLayout({
 }: SkeletonLayoutProps) {
   const isMaxWidth = maxContentWidth === Number.MAX_VALUE || maxContentWidth === Number.MAX_SAFE_INTEGER;
   const anyPanelOpen = navigationOpen || toolsOpen;
+  const isMobile = useMobile();
```

Comment:
There was a plan to avoid using `useMobile` hook in this code to allow server-side-rendering. Is it possible to keep mobile overrides with CSS media queries only?